### PR TITLE
Update Help links

### PR DESCRIPTION
**Pull request by olegrise**
@alex-deaconu please review.

### CODE REVIEW

**Comment by ezequielc** [comment]
@olegrise Alex is on vacation for the next week. You can tag me or @fjvallarino for the reviews.
Do you have a staged version for this change? Thanks

**Comment by olegrise** [comment]
@ezequielc good to know. Staged version https://store-stage-0.risevision.com

File: test/e2e/test-suite.scenarios.js [hunk 1]
```diff
@@ -28,21 +28,21 @@
       expect(element(by.id("coreApiUrl")).getText()).to.eventually.contain("rvacore-test.appspot.com");
     });
     
-    var registrationScenarios = new RegistrationScenarios();
+    // var registrationScenarios = new RegistrationScenarios();
```
Comment (ezequielc) [inline test/e2e/test-suite.scenarios.js]: @olegrise I think you unintentionally commented these test scenarios.  Thanks

Comment (olegrise) [inline test/e2e/test-suite.scenarios.js]: @ezequielc you are right. I'm glad you've noticed. Thanks.

**Comment by olegrise** [comment]
@ezequielc please review the last commit.

**Comment by ezequielc** [comment]
@olegrise Looks good 👍 Thanks!

**Pull request merged**